fmriprep/interfaces/cifti.py

```diff
@@ -117,122 +135,146 @@ def _fetch_data(self):
         download_link = getters.OSF_PROJECT_URL + getters.OSF_RESOURCES[label_space][0]
         return annotation_files, label_file, download_link
 
-
-def create_cifti_image(bold_file, label_file, annotation_files, gii_files,
-                       volume_target, surface_target, tr, download_link=None):
+    def _create_cifti_image(bold_file, label_file, annotation_files, gii_files,
```

Comment:
This either needs to be a `@staticmethod` or have `self` as an initial parameter.